docs/admin/config/deployment-environments.md

```diff
@@ -162,34 +159,35 @@ data:
 :sync: cluster-existing
 
 ```python
-from nemo_curator.utils.distributed_utils import get_client
+import ray
+from nemo_curator.core.client import RayClient
 
-# Connect to existing scheduler
-client = get_client(scheduler_address="tcp://scheduler:8786")
+# Connect to existing Ray cluster
+ray.init(address="ray://head-node:10001")
 
-# Using scheduler file (common in Slurm)
-client = get_client(scheduler_file="/shared/scheduler.json")
+# Connect using RAY_ADDRESS environment variable
+ray.init()  # Uses RAY_ADDRESS if set
```

Comment:
i don't think we should suggest this, just `RayClient`

cc @ayushdg as you've looked at slurm stuff a bit